test/FillWithSameByte.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.8;
+
+library FillWithSameByte {
```

Comment:
I like that! I think its good practice. Maybe on eco-system Friday, we can make a PR to foundry to add it by default.